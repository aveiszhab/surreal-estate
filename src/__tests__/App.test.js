import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter as Router } from "react-router-dom";
import App from "../components/App";

describe("App", () => {
  it("renders correctly to match the App snapshot", () => {
    const { asFragment } = render(
      <Router>
        <App />
      </Router>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
