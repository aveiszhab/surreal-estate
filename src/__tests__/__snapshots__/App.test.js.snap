// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly to match the App snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="navbar"
    >
      <img
        alt="logo"
        class="navbar-logo"
        src="MyLogo.png"
      />
      <h2>
        Surreal Estate
      </h2>
      <ul
        class="navbar-links"
      >
        <li
          class="navbar-links-item"
        >
          <a
            href="/"
          >
            View Properties
          </a>
        </li>
        <li
          class="navbar-links-item"
        >
          <a
            href="/add-property"
          >
            Add Property Page
          </a>
        </li>
      </ul>
    </div>
    <div
      class="full-page"
    >
      <div
        class="sidebar"
      >
        <div>
          <h2>
            Filter by city
          </h2>
          <div
            class="sidebar-links"
          >
            <div
              class="sidebar-links-items"
            >
              <a
                href="/?query={\\"city\\":\\"Manchester\\"}"
              >
                Manchester
              </a>
            </div>
            <div
              class="sidebar-links-items"
            >
              <a
                href="/?query={\\"city\\":\\"Leeds\\"}"
              >
                Leeds
              </a>
            </div>
            <div
              class="sidebar-links-items"
            >
              <a
                href="/?query={\\"city\\":\\"Sheffield\\"}"
              >
                Sheffield
              </a>
            </div>
            <div
              class="sidebar-links-items"
            >
              <a
                href="/?query={\\"city\\":\\"Liverpool\\"}"
              >
                Liverpool
              </a>
            </div>
          </div>
          <h2>
            Sort by Price
          </h2>
          <div
            class="sidebar-links"
          >
            <div
              class="sidebar-links-items"
            >
              <a
                href="/?sort={\\"price\\":1}"
              >
                Price Ascending
              </a>
            </div>
            <div
              class="sidebar-links-items"
            >
              <a
                href="/?sort={\\"price\\":-1}"
              >
                Price Descending
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="properties"
      />
    </div>
  </div>
</DocumentFragment>
`;
